moveit_core/kinematics_base/include/moveit/kinematics_base/kinematics_base.h

```diff
@@ -388,6 +391,8 @@ class KinematicsBase
    * @param tip_frame The tip of the chain
    * @param search_discretization The discretization of the search when the solver steps through the redundancy
    * @return True if initialization was successful, false otherwise
+   *
+   * Note: this is intended to be removed in future ROS versions in favor of robot_model being passed in
```

Comment:
I tried adding MOVEIT_DEPRECATED to both this function and the other version of initialize() that uses the string urdf below. gcc spits out tons of warnings though, complaining about the below deprecated initialize() calling this deprecated initialize. This is on ubuntu 16.04 with `gcc version 5.4.0 20160609 (Ubuntu 5.4.0-6ubuntu1~16.04.2) `.